bin/cheritest/cheritest_ccall.c

```diff
@@ -165,7 +165,7 @@ void
 test_fault_creturn(const struct cheri_test *ctp __unused)
 {
 
-	CHERI_CRETURN();
+	__asm__ __volatile__("creturn" : : : "memory");
```

Comment:
I'm slightly concerned about adding MD code here, but it's probably just a one-liner on every architecture so not a big deal and worth it to kill off the whole set of macros.